### ISSUE TITLE
delete function need the first argument must be a string or Buffer: ERROR

### ISSUE THREAD

**Issue by radjivC** [visionmedia/supertest#335]
Hey everyone I'm trying to use the delete function,

I receive this error

```
[TypeError: first argument must be a string or Buffer]
error: unhandledRejection reason=first argument must be a string or Buffer, currentUserEmail=undefined, currentRequest=undefined
double callback!
double callback!
double callback!
```

my code

```
      request(app).delete("/1/goat/")
      .send({
        token: usert.token
        goat_id: goat_id
      }) 
```

which I don't understand because the token is a string.

Thanks for your help.

**Comment by alphaaddynamo** [comment]
Did you manage to get past this? I inspected the incoming `body` in my app and the `.send` in supertest's agent doesn't seem to be sending the `body` i specified.

**Comment by radjivC** [comment]
@alphaaddynamo  Yes, add a .query

```
request(app).delete("/1/claim").query({
  token: token,
  goat_id: goat_id
})

```

**Comment by alphashuro** [comment]
I found out they didn't support send so I submitted a PR to superagent and supertest and they are both merged now, you can use supertest 3.0.0 if you want to use . send with a delete method

**Comment by radjivC** [comment]
@alphashuro Good job mate. :)
